src/python_routing_v02/troute/routing/fast_reach/mc_reach.pyx

```diff
@@ -902,76 +937,82 @@ cpdef object compute_network_structured_obj(
 
     #Run time
     while timestep < nsteps+1:
-      for r, reach_type in reach_objects:
+        for r, reach_type in reach_objects:
 
-        #Need to get quc and qup
-        upstream_flows = 0.0
-        previous_upstream_flows = 0.0
-        for id in r.upstream_ids: #Explicit loop reduces some overhead
-          upstream_flows += flowveldepth[id, timestep, 0]
-          previous_upstream_flows += flowveldepth[id, timestep-1, 0]
+            #Need to get quc and qup
+            upstream_flows = 0.0
+            previous_upstream_flows = 0.0
+            for id in r.upstream_ids: #Explicit loop reduces some overhead
+                upstream_flows += flowveldepth[id, timestep, 0]
+                previous_upstream_flows += flowveldepth[id, timestep-1, 0]
```

Comment:
Will you confirm that this is exclusively a formatting update?

Reply:
Should be. We moved some indentations that were half indented around in some places to make it look cleaner. Still works.